### PR TITLE
Fix typo on todo Sqlalchemy tutorial

### PR DESCRIPTION
**Pull request by su-shubham**

### CODE REVIEW

File: docs/tutorials/sqlalchemy/0-introduction.rst [hunk 1]
```diff
@@ -96,8 +96,8 @@ serializable by Litestar.
 Behavior
 ++++++++
 
-the ``add_item()`` and ``update_item()`` routes no longer return the full collection, instead they return the item that
-was added or updated. This is a minor detail change, but it is worth noting as it brings the behavior of the app closer
+The ``add_item()`` and ``update_item()`` routes no longer return the full collection, instead they return the item that
+was added or updated. This is a minor detail change, but it is worth nothing as it brings the behavior of the app closer
```
Comment (cofin) [inline docs/tutorials/sqlalchemy/0-introduction.rst:100]: I think we actually mean `noting` here and not `nothing`.  Maybe it should be `it is worth mentioning`

Comment (su-shubham) [reply docs/tutorials/sqlalchemy/0-introduction.rst]: Thankyou @cofin. Review latest changes.

**Review by cofin — APPROVED** [review]

**Comment by cofin** [comment]
@all-contributors please add @su-shubham for docs

**Review by JacobCoffee — APPROVED** [review]

**Comment by cofin** [comment]
@JacobCoffee Any idea why these unrelated tests are failing?

**Comment by JacobCoffee** [comment]
@cofin I think these are upstream (read: our main branch) changes, because the Pydantic v2 branch is also failing due to the same thing, so this can be merged ig

**Pull request merged**